source/Server.Contracts/KnownVariables.cs

```diff
@@ -40,6 +40,56 @@ public static string Name(string variableName)
             {
                 return $"{variableName}.Name";
             }
+            
+            public static string Type(string variableName)
+            {
+                return $"{variableName}.Type";
+            }
+            
+            public static string Thumbprint(string variableName)
+            {
+                return $"{variableName}.Thumbprint";
+            }
+            
+            public static string RawOriginal(string variableName)
+            {
+                return $"{variableName}.RawOriginal";
+            }
+            
+            public static string Password(string variableName)
+            {
+                return $"{variableName}.Password";
+            }
+            
+            public static string Pfx(string variableName)
+            {
+                return $"{variableName}.Pfx";
+            }
+            
+            public static string CertificateValue(string variableName)
+            {
+                return $"{variableName}.Certificate";
```

Comment:
Does it matter here that the method name does not match the string variable?

Reply:
This was to fix a compile error as the method could not have the same name as the class.